### PR TITLE
Updated create topics to utilize the new Java AdminClient instead of …

### PR DESCRIPTION
**Pull request by mkwhitacre**
…legacy AdminUtils.

The conversion to use the new AdminClient isn't complete by any stretch but was just starting to play around with how to move off the Scala version.

### CODE REVIEW

File: common-kafka-admin/src/main/java/com/cerner/common/kafka/admin/KafkaAdminClient.java [hunk 1]
```diff
@@ -500,6 +513,8 @@ public void createTopic(String topic, int partitions, int replicationFactor, Pro
 
         if (!operationCompleted)
             throw new AdminOperationException("Timeout waiting for topic " + topic + " to be created");
+
+        updateTopicConfig(topic, topicConfig);
```
Comment (bbaugher) [inline common-kafka-admin/src/main/java/com/cerner/common/kafka/admin/KafkaAdminClient.java]: Looks like you can supply the configuration when you create the topic [with NewTopic.configs(Map)](https://kafka.apache.org/20/javadoc/org/apache/kafka/clients/admin/NewTopic.html#configs-java.util.Map-)

Comment (mkwhitacre) [reply common-kafka-admin/src/main/java/com/cerner/common/kafka/admin/KafkaAdminClient.java]: Updated: https://github.com/cerner/common-kafka/pull/16/commits/459298fe7ae187062df754bf280f8e3ed99220c7

**Comment by mkwhitacre** [comment]
@bbaugher and @noslowerdna what are your thoughts on continuing to work through this?  I know with #17 this stuff is now deprecated but is it worth still refactoring the old API on new stuff?

**Comment by bbaugher** [comment]
If it works without changing things I'd probably say just don't do it

**Comment by mkwhitacre** [comment]
The counter argument is if we do it, we'd get to drop some hefty dependencies like Scala if we can move farther off of those APIs.

**Comment by noslowerdna** [comment]
@mkwhitacre this change LGTM. I think the refactoring could be a worthwhile initial step since we will at some point need to understand how to directly use the new API anyway when a wrapper admin client is no longer supplied by common-kafka, and the changes here could be a useful reference for what that replacement code looks like.

**Pull request merged**